docs/faq/non-mc-projects.md

```diff
@@ -1,6 +1,6 @@
 # Are non-Minecraft projects allowed?
 
-Yes.
+**Short Answer:** Yes
 
-CodeMC accepts a wide range of Projects for all kinds of topics, games, etc.  
-Only important part is, that the project in question does not violate our [Usage Guidelines](../usage-guidelines.md) to be allowed.
+**Long Answer:** CodeMC accepts any Java project that shows a level of quality and usefulness for the overall Java Community, be it a library, a software or even a game.  
```

Comment:
I feel excluded as a Kotlin user.

In all seriousness, I would implicitly include projects that are related to Java instead of explicitly saying "Java Community" to include projects that are technically "Java Projects" but don't have Java sources (e.g. Kotlin, Groovy, Scala, Clojure, etc). Also includes minor Grammarly changes.
```suggestion
**Long Answer:** CodeMC accepts any Java (or Java-related) project that shows thorough quality and usefulness for the overall programming community, be it a library, software, or even a game. If you're using Kotlin or Scala, your project should compile to the JVM while continuing to provide services intended for the community.
```